drivers/gpu/drm/vc4/vc4_drv.h

```diff
@@ -351,6 +351,12 @@ struct vc4_hvs {
 	 * available.
 	 */
 	bool vc5_hdmi_enable_scrambling;
+
+	/*
+	 * 4096x2160@60 requires a core overclock to work, so register
+	 * whether that is sufficient.
+	 */
+	bool vc5_hdmi_enable_4096by2160
```

Comment:
Missing semi-colon.

Reply:
Indeed. gcc only gives a warning for this, which seems surprising.